JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/Services/IAuthenticationService.cs

```diff
@@ -7,7 +9,8 @@ public interface IAuthenticationService
     {
         bool IsAuthenticated { get; set; }
         UserModel CurrentUser { get; set; }
+        HttpClient _httpClient { get; set; }
         Task<bool> SignIn(LoginModel model);
-        void SignOut();
+        string SignOut();
```

Comment:
Please go back to void

Reply:
Need a result.  So we can create an attribute to check this or make this method as string/bool/int.